crates/cxx-qt-gen/src/generator/cpp/signal.rs

```diff
@@ -49,7 +49,7 @@ struct Parameters {
 fn parameter_types_and_values(
     parameters: &[ParsedFunctionParameter],
     type_names: &TypeNames,
-    self_ty: &str,
+    self_ty: &Ident,
```

Comment:
wonder if this should be called something else, if it is an Ident then why is it a `_ty` ? Maybe `self_ident` but not sure what the actual values coming in here are without digging around.

Or is this simply just `qobject_ident` ?